OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -330,6 +330,20 @@ public void FrameUpdate(FrameEventArgs frameEventArgs) {
         } else {
             string elementId = split[0];
 
+            // ":[element]" returns the default element of that type
+            if (elementId[0] == ':') {
+                switch (elementId[1..]) {
```

Comment:
You were worried about the string operation, right? You could just put `:` at the beginning of each case instead